### DISCUSSION TITLE
"Apparent" Altitude

### DISCUSSION THREAD

**Discussion by yugo76** [Stellarium/stellarium#3292] [General]
I am studying celestial navigation and I need to know if the "apparent" altitude of a celestial body is already the sight corrected for Hs (height of sextant or observer) index error and dip. In other words, Stellarium already gives the Ha directly. If this is the case, I would like to suggest a mode or a plug-in that simulates a sextant. There is the mouse pointer coordinates and the Nav stars but one that truly would simulate a sextant including "rocking the sun" feature.

**Comment by gzotti** [comment]
Such plugin MUST be created by someone familiar with the process. 

Please always include an explanation for non-navigators of what you mean e.g. with Ha. 

The apparent altitude as defined in astronomy is the vertical angular distance from the mathematical horizon and includes atmospheric refraction, nutation, precession, aberration and proper motion. There is no dip correction (see endless debates about "move horizon down when raising elevation"), and ... hm, I currently have forgotten what index error was, if not an instrument error.

For operational simulation of a sextant, I suggest using a game engine for serious gaming, and optionally combining this with Stellarium. See https://github.com/Stellarium/stellarium-unity for an example.
